comixed-importer/src/main/java/org/comixed/importer/ImportFileProcessor.java

```
@@ -41,19 +45,34 @@
 @ConfigurationProperties(value = "processor.file.import")
 public class ImportFileProcessor {
     private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
     protected List<PathReplacement> replacements = new ArrayList<>();
+
+    protected ComiXedUser importUser;
+
+    protected Map<String, String> currentPages = new HashMap<>();
+
     @Autowired
     private ComicRackBackupAdaptor backupAdaptor;
+
     @Autowired
     private ComicFileImportAdaptor importAdaptor;
 
+    @Autowired
+    private ComiXedUserRepository userRepository;
+
     public void setReplacements(List<String> replacements) {
         this.logger.debug("Processing {} replacement rules", replacements.size());
         for (String rule : replacements) {
             this.replacements.add(new PathReplacement(rule));
         }
     }
 
+    public void setImportUser(String user) {
+        this.logger.debug("Setting import user {}", user);
+        this.importUser = this.userRepository.findByEmail(user);
```

I think we should throw an exception here or in the caller if the user isn't found, to stop the import from running since that's a required element.